lib/param-parser.js

```diff
@@ -56,7 +56,12 @@ function parseFormDataParams (req, res, next) {
 
       if (param.type === "file") {
         // Validate the file (min/max size, etc.)
-        req.files[param.name] = parseParameter(param, req.files[param.name], param);
+        let file;
+        if (req.files[param.name] && req.files[param.name][0]) {
+          file = req.files[param.name][0];
```

Comment:
This weird let hand-off is so that we wind up with an undefined in the "right" cases for parseParameter so it validates empty values as expected.